tsconfig.json

```diff
@@ -5,6 +5,7 @@
     "lib": ["es2016", "es2017", "dom"],
     "moduleResolution": "node",
     "noImplicitAny": false,
+    "skipLibCheck": true,
```

Comment:
This options is required because the latest `htm` has broken type definitions.

I would recommend to add this option disregardless of the `htm` issue because I believe type checking of dependencies should be done in the dependencies themselves.

Reply:
Disagree :-1:. The library user would expect strict typing well-confirmed in our library.

I've already worked for fix `htm` in https://github.com/developit/htm/pull/153. Please wait for the next release of `htm`, or lock version to `3.0.1`.

Reply:
Thanks! So I will use `3.0.1` right now.